clients/include/utility.hpp

```diff
@@ -6043,6 +6043,35 @@ class Arguments
     }
 };
 
+inline void missing_file_error_message(const char* filename)
+{
+    std::cerr << "#" << std::endl;
+    std::cerr << "# error:" << std::endl;
+    std::cerr << "# cannot open file '" << filename << "'" << std::endl;
+    std::cerr << "#" << std::endl;
+    std::cerr << "# PLEASE READ CAREFULLY !" << std::endl;
+    std::cerr << "#" << std::endl;
+    std::cerr << "# What could be the reason of this error: " << std::endl;
+    std::cerr << "# You are running the testing application then it expects to find the file "
+                 "at the specified location. This means that either you did not download the test "
+                 "matrices or you did not specified the location of the folder containing your "
+                 "files. If you want to specify the location of the folder containing your files "
+                 "then you will find the needed information with 'hipsparse-test --help'."
+                 "If you need to download matrices, a cmake script "
+                 "'hipsparse_clientmatrices.cmake' is available from the hipsparse client package."
+              << std::endl;
+    std::cerr << "#" << std::endl;
+    std::cerr
+        << "# Examples: 'hipsparse_clientmatrices.cmake -DCMAKE_MATRICES_DIR=<path-of-your-folder>'"
+        << std::endl;
+    std::cerr << "#           'hipsparse-test --matrices-dir <path-of-your-folder>'" << std::endl;
```

Comment:
Just double checking, does hipsparse also have the --matrices-dir option?

Reply:
check hipsparse-test --help